<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-UICAQ40" failures="1" tests="3" name="Test" time="6.427" errors="0" timestamp="29 Sep 2022 15:13:08 GMT">
  <testcase classname="JenkinsTest.Test3" name="t3" time="3.117"/>
  <testcase classname="JenkinsTest.Test2" name="t3" time="1.647">
    <failure type="java.lang.AssertionError" message="expected [1] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [2]
	at JenkinsTest.Test2.t3(Test2.java:30)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- t3 -->
  <testcase classname="JenkinsTest.Test1" name="t2" time="1.65"/>
</testsuite> <!-- Test -->
